scripts/cluster/kubeadm.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: kubeadm.k8s.io/v1beta2
```

Comment:
@HongyuHe I realize you copied this yaml from your local setup. Can you please explain how it is different from the default one?

Reply:
ok, this file is deleted in the next commit